jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -650,7 +656,9 @@ public static String formatNoSuchMethod(String methodSig, Class<?> clazz1, Strin
 				return methodSig + "(loaded from " + callingClassInfo[0] + " by " + callingClassInfo[1] + ") called from " + clazz2.toString() +
 						" (loaded from " + calledClassInfo[0] + " by " + calledClassInfo[1] + ")";
 			} catch (Exception | VirtualMachineError e2) {
+				/*[IF JAVA_SPEC_VERSION < 20]*/
 				// don't catch ThreadDeath
+				/*[ENDIF] JAVA_SPEC_VERSION < 20 */
```

Comment:
It's not clear there's value to this comment: perhaps we should just delete it (from all versions).

Reply:
Works for me. Updated.